packages/contracts/contracts/SmartEscrowMono.sol

```diff
@@ -228,18 +228,16 @@ contract SmartEscrowMono is Context, IArbitrable, ReentrancyGuard {
     uint256 providerAward,
     bytes32 details
   ) external nonReentrant {
-    // resolver splits locked deposit remainder between client & provider(s)
+    // called by lex dao
     Locker storage locker = lockers[index];
     require(locker.resolverType == ADR.LEX_DAO, "!lex");
+    require(locker.locked, "!locked");
+    require(locker.total > locker.released, "released");
+    require(_msgSender() == locker.resolver, "!resolver");
 
     uint256 remainder = locker.total.sub(locker.released);
     uint256 resolutionFee = remainder.div(20); // calculates dispute resolution fee (5% of remainder)
 
-    require(locker.locked, "!locked");
-    require(locker.total > locker.released, "released");
-    require(_msgSender() == locker.resolver, "!resolver");
-    require(_msgSender() != locker.client, "resolver == client");
-    require(_msgSender() != locker.provider, "resolver == provider");
```

Comment:
> lines 241 and 242: if _msgSender() == locker.resolver, then do we need to check if its the client or provider?
same with lines 275 and 276

I've removed these checks. 
They existed in the older version, so kept them. 
But I agree that they seem redundant.